### ISSUE TITLE
Vimeo embedding inconsistent

### ISSUE THREAD

**Issue by laryn** [backdrop-contrib/video_embed_field#12]
I'm having a strange result where embedding Vimeo seems to work fine on one site but not another. The videos have some domain-level privacy controls set on Vimeo, though I'm not sure that's the issue here. Strangely, I did find that changing the embed code to force SSL seems to solve it. Does that seem reasonable?

**Comment by laryn** [comment]
Closing for now, I don't even remember which site this was on.

**Comment by olafgrabienski** [comment]
> changing the embed code to force SSL seems to solve it. Does that seem reasonable?

Interesting! @laryn Was the site in question maybe using the [GDPR Cookies](https://github.com/backdrop-contrib/gdpr_cookies) module? I'm asking because on one of my sites where I use *GDPR Cookies* and its Vimeo vanisher, in a certain situation the video doesn't work, and there appears an "embed code not valid" message. See my (closed) [issue report in the GDPR Cookies queue](https://github.com/backdrop-contrib/gdpr_cookies/issues/22).

Forcing the embed code to use SSL, like you did in https://github.com/backdrop-contrib/video_embed_field/pull/13, fixes the issue for me.

**Comment by olafgrabienski** [comment]
@laryn I saw that you merged #34 but even if the PR title is promising, it didn't fix the Vimeo issue.

In line 574 the https protocol is missing from the player URL:

```
    '#markup' => '<iframe' (...) src="//player.vimeo.com/video/' . $id .
```

Adding `https:` like below works for me.

```
    '#markup' => '<iframe' (...) src="https://player.vimeo.com/video/' . $id .
```
